DUI3-DX/Sdk/Speckle.Connectors.Utils/Operations/ReceiveOperation.cs

```diff
@@ -32,26 +30,17 @@ public ReceiveOperation(IHostObjectBuilder hostObjectBuilder)
   )
   {
     // 2 - Check account exist
-    Account account =
-      AccountManager.GetAccounts().FirstOrDefault(acc => acc.id == accountId)
-      ?? throw new SpeckleAccountManagerException();
+    Account account = AccountManager.GetAccount(accountId);
 
     // 3 - Get commit object from server
-    Client apiClient = new(account);
-    ServerTransport transport = new(account, projectId);
```

Comment:
interesting that we never disposed of the transport. Could this have been the cause of some of our websocket issues?

Reply:
Potentially, although, I'm not going to pretend I understand our web socket issue 🤣